_docs-2-0/development/client-properties.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Client Properties
+category: development
+order: 9
+---
+
+<!-- WARNING: Do not edit this file. It is a generated file that is copied from Accumulo build (from core/target/generated-docs) -->
+
+Below are properties set in `accumulo-client.properties` that configure Accumulo clients:
+
+| Property | Default value | Description |
```

Comment:
This could be updated with since tag column

Reply:
Fixed in de495ff2c